commands/homepage.js

```diff
@@ -0,0 +1,10 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
const { SlashCommandBuilder } = require("@discordjs/builders");
```